### DISCUSSION TITLE
GREEN LED LIGHT!!!

### DISCUSSION THREAD

**Discussion by hecticdot** [OnionUI/Onion#1417] [Ideas]
If there’s one thing I want most to happen (if possible) is a way to dim or turn off that damn green led light though the OS. Playing at night in bed would be a great without my bedroom being lit up by green. Plz look into it. . I beg u onion os team youre my only hope to defeat this green menace.  also onion os slaps Thx.

**Comment by thomasklemenc** [comment]
Not possible, sorry. Cover it using some black tape or remove it from the board.
